packages/desktop-gui/src/settings/file-preference_spec.jsx

```diff
@@ -8,11 +8,11 @@ import '../main.scss'
 /* global cy, Cypress */
 describe('FilePreference', () => {
   const availableEditors = [
-    { id: 'atom', name: 'Atom', isOther: false, openerId: 'atom' },
```

Comment:
`openerId` and `binary` are the same, but duplicated. Removing this is technical a breaking change, so now is the time to do it. Worst case scenario is the user will need to select their preferred editor again.